### PR TITLE
AT_003.002 | Freestyle project > Rename with invalid data

### PR DESCRIPTION
**Pull request by pojirakula**
Approve, please

### CODE REVIEW

**Comment by pojirakula** [comment]
[AT_003.002 | Freestyle project > Rename with invalid data](https://trello.com/c/ZwOHfRd7/669-at003002-freestyle-project-rename-with-invalid-data)

File: src/test/java/FreestyleProjectRenameWithInvalidData.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        getDriver().switchTo().alert().accept();
+
+    }
+
+    private void clearSendClick(String newProjectName) {
+        getDriver().findElement(By.xpath("//input[@checkdependson='newName']")).clear();
+        getDriver().findElement(
+                By.xpath("//input[@checkdependson='newName']")).sendKeys(newProjectName);
+        getDriver().findElement(By.xpath("//button[@type='submit']")).click();
+
```
Comment (anonymoustuna) [inline src/test/java/FreestyleProjectRenameWithInvalidData.java:60]: No need in space here.

File: src/test/java/FreestyleProjectRenameWithInvalidData.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        getDriver().findElement(By.xpath("//a[normalize-space(text())='Dashboard']")).click();
+
+        Actions action = new Actions(getDriver());
+        action.moveToElement(getDriver().findElement(
+                By.xpath("//a[@href='job/" + newProjectName + "/']"))).click().build().perform();
+
+        getDriver().findElement(By.xpath("//span[text()='Delete Project']")).click();
+
+        getDriver().switchTo().alert().accept();
+
```
Comment (anonymoustuna) [inline src/test/java/FreestyleProjectRenameWithInvalidData.java:52]: No need in space here.

**Review by anonymoustuna — APPROVED** [review]

**Review by GitForProjects — APPROVED** [review]
looks good

File: src/test/java/FreestyleProjectRenameWithInvalidData.java (hunk 2 above)
Comment (pojirakula) [reply src/test/java/FreestyleProjectRenameWithInvalidData.java:52]: Removed. Thanks

File: src/test/java/FreestyleProjectRenameWithInvalidData.java (hunk 1 above)
Comment (pojirakula) [reply src/test/java/FreestyleProjectRenameWithInvalidData.java:60]: ok

**Pull request merged**